src/v3/components/date_picker/index.tsx

```diff
@@ -0,0 +1,141 @@
+import { uk } from 'date-fns/locale';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+import { useCallback, useEffect, useState } from 'react';
+import { addYears, getWeeksInMonth, startOfYear, subYears } from 'date-fns';
+import Stack from '@mui/material/Stack';
+import { format } from 'date-fns';
+import { ClickAwayListener } from '@mui/material';
+import { Button, Typography } from '@components';
+import { DesktopDatePicker } from '@mui/x-date-pickers';
+import '../../global.css';
+import React from '@services/react';
+import ButtonField from './view/button';
+import { CPEDatePickerProps } from './date_picker.types';
+import DatePickerInputField from './view/input';
+import {
+  StyleDatePickerActionBar,
+  StyleDatePickerDay,
+  StyleDatePickerDesktopPaper,
+  StyleDatePickerLayout,
+  StyleDatePickerPopper,
+  StyleDatePickerToolbar,
+} from './date_picker.style';
+
+const CPEDatePicker = ({ initDate = null, view, label, disablePast, limitYear = false }: CPEDatePickerProps) => {
+  const [open, setOpen] = useState<boolean>(false);
+  const [value, setValue] = useState<Date | null>(null);
+  const [innerValue, setInnerValue] = useState<Date | null>(initDate);
+
+  const [height, setHeight] = useState('300px'); // Initial height
+
+  const changeHeight = useCallback((event) => {
+    if (getWeeksInMonth(new Date(event), { locale: uk, weekStartsOn: 1 }) === 6) setHeight('340px');
+    else setHeight('300px');
+  }, []);
+
+  const handleClickAway = () => {
+    if (open) setOpen(false);
+  };
+
+  useEffect(() => {
+    if (initDate === null && open) setInnerValue(new Date());
+  }, [open]);
+
+  const viewProps = view === 'button' ? { field: ButtonField } : { textField: DatePickerInputField };
+
+  return (
+    <ClickAwayListener onClickAway={handleClickAway}>
+      <div>
+        <LocalizationProvider dateAdapter={AdapterDateFns} adapterLocale={uk}>
```

Comment:
@eloyEntony: will this make this component to always use `uk` as default? If default is set, let’s make it english.